service/kbupd/build.rs

```diff
@@ -17,6 +17,26 @@ fn main() {
 
     let mut protoc = prost_build::Config::new();
     protoc
+        .type_attribute(".", "#[derive(serde::Serialize)]")
+        .field_attribute("node_id", "#[serde(with = \"serde_with::As::<serde_with::hex::Hex>\")]")
+        .field_attribute(
+            "EnclaveFrontendPartitionStatus.group_id",
+            "#[serde(with = \"serde_with::As::<serde_with::hex::Hex>\")]",
```

Comment:
I notice [`kbupd_config` uses  `kbupd_util::hex::SerdeFixedLengthHex`](https://github.com/mobilecoinofficial/SecureValueRecovery/blob/380d71f6ca6f086bd2be2834be60fb09f58bdd84/service/kbupd_config/src/replica/config.rs#L32), so let's use that instead of `serde_with`